src/fitnesseMain/FitNesseMain.java

```diff
@@ -49,6 +49,32 @@ public static void main(String[] args) {
   protected static void exit(int exitCode) {
     System.exit(exitCode);
   }
+  
+  // Testing code anaylsis tools
+  public String testMethod(){
+    
+  }
+  
+  public String badCommaWhiteSpaceBeforeAndAfter(String input ,String anotherInput)
+    {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') 'method def modifier' has incorrect indentation level 2, expected level should be 4. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.cloudmok.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Class 'FitNesseMain' looks like designed for extension (can be subclassed), but the method 'badGenericsWhiteSpace' does not have javadoc that explains how to do that safely. If class is not designed for extension consider making the class 'FitNesseMain' final or making the method 'badGenericsWhiteSpace' static/final/abstract/empty, or adding allowed annotation for the method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.cloudmok.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.design.DesignForExtensionCheck)